code/datums/craft/recipes/tools.dm

```diff
@@ -94,8 +94,8 @@
 	)
 
 /datum/craft_recipe/tool/toolimplant
-	name = "Multitool implant"
-	result = /obj/item/organ_module/active/multitool/improvised
+	name = "Improvised multitool implant"
+	result = /obj/item/organ_module/active/simple/makeshift
 	steps = list(
```

Comment:
While you're at it, consider simplifying that craft recipe, it's unreasonably complicated and long.

Reply:
Removing the robotic arm would cut down on a lot of the length, maybe the circuit board and cable coil as well?

Reply:
Your call.